package.json

```
@@ -1,7 +1,8 @@
 {
   "name": "@nuxt/ui",
   "version": "2.10.0",
-  "repository": "https://github.com/nuxt/ui",
+  "repository": "nuxt/ui",
+  "homepage": "https://image.nuxt.com",
```

shouldn't it be
```suggestion
  "homepage": "https://ui.nuxt.com",
```
?